### ISSUE TITLE
404 error on some external resources

### ISSUE THREAD

**Issue by sandrobonazzola** [oVirt/ovirt-site#2842]
#### Type of problem?
404

enabling external site test, htmlproofer reports:

```
- ./_site/develop/release-management/features/node/node-next-persistence.html
  *  External link http://rpm.org/devel_doc/plugins.html failed: 404 No error
- ./_site/develop/release-management/features/virt/iothreads-support.html
  *  External link http://wiki.mikejung.biz/KVM_/_Xen failed: 404 No error
- ./_site/develop/release-management/process/press-plan.html
  *  External link http://www.informationweek.com/authors/7032 failed: 404 No error
  *  External link http://www.informationweek.com/authors/Charles-Babcock failed: 404 No error
  *  External link https://www.infoworld.com/author/Paul-Venezia/ failed: 404 No error
- ./_site/documentation/administration_guide/index.html
  *  External link https://ovirt.org/documentation/installing_red_hat_virtualization_as_a_self-hosted_engine_using_the_command_line/index#Enabling_the_RHVH_repository_SHE_cli_deploy failed: 404 No error
  *  External link https://ovirt.org/documentation/installing_red_hat_virtualization_as_a_self-hosted_engine_using_the_command_line/index#Enabling_the_Red_Hat_Enterprise_Linux_Host_Repositories_SHE_cli_deploy failed: 404 No error
- ./_site/documentation/doc-Product_Guide/index.html
  *  External link https://ovirt.org/documentation/java_sdk_guide/index# failed: 404 No error
  *  External link https://ovirt.org/documentation/python_sdk_guide/index# failed: 404 No error
  *  External link https://ovirt.org/documentation/technical_reference/index# failed: 404 No error
- ./_site/documentation/doc-Technical_Notes/RHSA-20211169-06.html
  *  External link https://errata.devel.redhat.com/advisory/68228 failed: 401 No error
- ./_site/documentation/doc-Technical_Notes/RHSA-20214626-06.html
  *  External link https://errata.devel.redhat.com/advisory/81159 failed: 401 No error
- ./_site/documentation/migrating_from_a_standalone_manager_to_a_self-hosted_engine/index.html
  *  External link https://ovirt.org/documentation/installing_red_hat_virtualization_as_a_self-hosted_engine_using_the_command_line/index#ref-Recommended_Settings_for_Multipath_conf_SHE_cli_deploy failed: 404 No error
- ./_site/documentation/upgrade_guide/index.html
  *  External link https://ovirt.org/documentation/installing_red_hat_virtualization_as_a_standalone_manager_with_remote_databases/index#Installing_and_Configuring_Data_Warehouse_on_a_Separate_Machine_install_RHVM failed: 404 No error
  *  External link https://ovirt.org/documentation/installing_red_hat_virtualization_as_a_standalone_manager_with_remote_databases/index#Preparing_a_Remote_PostgreSQL_Database_install_RHVM failed: 404 No error
- ./_site/documentation/virtual_machine_management_guide/index.html
  *  External link https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/administration_guide/sect-hosts_and_networking#setting-up-and-configuring-sr-iov failed: 404 No error
- ./_site/release/4.0.5/index.html
  *  External link https://access.redhat.com/documentation/en/red-hat-virtualization/4.0/paged/self-hosted-engine-guide/ failed: 404 No error
  *  External link https://access.redhat.com/documentation/en/red-hat-virtualization/4.0/paged/upgrade-guide/ failed: 404 No error

```

**Comment by apinnick** [comment]
I can have a look at some of these.

**Comment by apinnick** [comment]
This earlier PR fixed the VMM guide: https://github.com/oVirt/ovirt-site/pull/2845

**Comment by apinnick** [comment]
@sandrobonazzola Please add documentation/doc-Technical_Notes and documentation/doc-Product_Guide to the "do not build" list.

**Comment by sandrobonazzola** [comment]
Updated list:
```
- ./_site/develop/release-management/features/network/networkfilterparameters.html
  *  External link https://github.com/libvirt/libvirt/blob/master/docs/schemas/nwfilter.rng failed: 404 No error
```
tracked in #2857 

```
- ./_site/documentation/administration_guide/index.html
  *  External link https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/managing_hosts/host-management-without-goferd-and-katello-agent_managing-hosts failed: 404 No error
  *  External link https://ovirt.org/documentation/installing_red_hat_virtualization_as_a_self-hosted_engine_using_the_command_line/index#Enabling_the_RHVH_repository_SHE_cli_deploy failed: 404 No error
  *  External link https://ovirt.org/documentation/installing_red_hat_virtualization_as_a_self-hosted_engine_using_the_command_line/index#Enabling_the_Red_Hat_Enterprise_Linux_Host_Repositories_SHE_cli_deploy failed: 404 No error
```
@oVirt/ovirt-documentation can you help here?

```
- ./_site/documentation/virtual_machine_management_guide/index.html
  *  External link https://github.com/oVirt/ovirt-vmconsole/blob/master/README#L116 failed: 404 No error
```
@ahadas @nyoxi @mz-pdm can you help here?

```
- ./_site/release/3.6.1/index.html
  *  External link https://docs.jboss.org/author/display/AS71/Security+Realms failed: 404 No error
  *  External link https://docs.jboss.org/author/display/WFLY8/CLI+Recipes failed: 404 No error
```
@arso @mwperina can you help here?

**Comment by nyoxi** [comment]
> `  *  External link https://github.com/oVirt/ovirt-vmconsole/blob/master/README#L116 failed: 404 No error`

The document has been moved. New link is `https://github.com/oVirt/ovirt-vmconsole/blob/master/README.md#customization`.

**Comment by sandrobonazzola** [comment]
Fixed via #2873

**Comment by sandrobonazzola** [comment]
This is still there:
```
- ./_site/release/3.6.1/index.html
  *  External link https://docs.jboss.org/author/display/AS71/Security+Realms failed: 404 No error
  *  External link https://docs.jboss.org/author/display/WFLY8/CLI+Recipes failed: 404 No error
```

**Comment by apinnick** [comment]
> This is still there:
> 
> ```
> - ./_site/release/3.6.1/index.html
>   *  External link https://docs.jboss.org/author/display/AS71/Security+Realms failed: 404 No error
>   *  External link https://docs.jboss.org/author/display/WFLY8/CLI+Recipes failed: 404 No error
> ```

@sandrobonazzola 
This is upstream only. It's coming from this file: source/release/3.6.1/index.md

**Comment by sandrobonazzola** [comment]
> @sandrobonazzola This is upstream only. It's coming from this file: source/release/3.6.1/index.md

Fixed in #2886
